analyzer/src/main/kotlin/managers/Bundler.kt

```diff
@@ -19,10 +19,46 @@
 
 package com.here.ort.analyzer.managers
 
+import ch.frankel.slf4k.error
+import ch.frankel.slf4k.info
+import ch.frankel.slf4k.warn
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.module.kotlin.readValue
 import com.here.ort.analyzer.PackageManager
 import com.here.ort.analyzer.PackageManagerFactory
-
+import com.here.ort.downloader.VersionControlSystem
+import com.here.ort.model.AnalyzerResult
+import com.here.ort.model.Identifier
+import com.here.ort.model.Package
+import com.here.ort.model.PackageReference
+import com.here.ort.model.Project
+import com.here.ort.model.RemoteArtifact
+import com.here.ort.model.Scope
+import com.here.ort.model.VcsInfo
+import com.here.ort.model.jsonMapper
+import com.here.ort.model.yamlMapper
+import com.here.ort.utils.ProcessCapture
+import com.here.ort.utils.log
 import java.io.File
+import java.io.IOException
+import java.nio.file.Files
+import java.nio.file.StandardCopyOption
+import java.util.SortedSet
+
+const val DEPS_LIST_RUBY = """#!/usr/bin/ruby
```

Comment:
I wonder whether we should actually commit this as a separate file under `resources`, so the file could be opened in an editor and we'd get proper Ruby syntax highlighting.